config/gradlew.bat

```diff
@@ -0,0 +1,100 @@
+@rem
+@rem Copyright 2015 the original author or authors.

```

Comment:
Shouldn't we add the copyrights as 2022?

Reply:
These gradle scripts belong to their respective authors and not to us.
When a new project is created in Android Studio (even in the most recent versions of it) these scripts still show up with 2015 copyrights.